src/demo-app/gestures/gestures-demo.scss

```diff
@@ -3,5 +3,5 @@
   width: 200px;
   background: gray;
   padding: 15px;
-  color: #FFF;
-}
\ No newline at end of file
+  color: white;
```

Comment:
:shrug: I meant it was smaller. I don't think it's that important after thinking about it.